javascript-objects/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Javascript Objects</title>
+  </head>
+  <body>
+    <script src="main.js">
+      var student = {
+        firstName: 'Alan',
+        lastName: 'Kim',
+        age: 24,
+        };
+
+      var fullName = student.firstName + ' ' + student.lastName
+      console.log('value of fullName:', fullName);
+
+      student.livesInIrvine = false;
+      console.log('value of livesInIrvine:', student.livesInIrvine);
+
+      student.previousOccupation = 'Lab Technician';
+      console.log('value of previousOccupation:',student.previousOccupation);
+
+      console.log('value of student:',student);
+
+      var vehicle = {
+        make: 'Hyundai',
+        model: 'Santa Fe',
+        year: '2016',
+      };
+
+      vehicle.color ='black';
+      console.log('value of vehicle["color"]:',vehicle['color']);
+
+      vehicle.isConvertible = false;
+      console.log('value of vehicle["isConvertible"]:',vehicle['isConvertible']);
+
+      console.log('value of vehicle:',vehicle);
+
+      var pet = {
+        name: 'Lucky',
+        type: 'Dog',
+      }
+
+      delete pet.name;
+      delete pet.type;
+      console.log('value of pet:',pet);
+
+    </script>
```

Comment:
This works well but instruction stated as following: `Create an index.html containing an HTML skeleton with a <script> tag that links the provided main.js file. Open index.html in your browser. You will be completing the rest of these steps within main.js.`

So next time, we need to create `main.js` file and link it from `index.html` file 👍🏼